config/samples/nodeaffinityscoring/multiarch_v1beta1_nodeaffinityscoring_clusterpodplacementconfig.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: multiarch.openshift.io/v1beta1
+kind: ClusterPodPlacementConfig
+metadata:
+  name: cluster
+spec:
+  logVerbosity: Normal
+  namespaceSelector:
+    matchExpressions:
+    - key: multiarch.openshift.io/exclude-pod-placement
+      operator: DoesNotExist
+  plugins:
+    nodeAffinityScoring:
+      enabled: true
+      platforms:
+      - architecture: amd64
+        # os: linux # TODO: this structure can support the os field in the future (e.g., for win containers)
```

```suggestion
        os: linux
```
Add it or don't.... leave the comment in the code.